plugin/output/kafka/kafka.go

```diff
@@ -188,6 +222,34 @@ func (p *Plugin) Stop() {
 
 func (p *Plugin) newProducer() sarama.SyncProducer {
 	config := sarama.NewConfig()
+	config.ClientID = "sasl_scram_client"
+	// kafka auth sasl
+	if p.config.SaslEnabled {
+		config.Net.SASL.Enable = true
+		config.Net.SASL.Handshake = true
+		config.Net.SASL.User = p.config.SaslUsername
+		config.Net.SASL.Password = p.config.SaslPassword
+		config.Net.SASL.SCRAMClientGeneratorFunc = func() sarama.SCRAMClient { return &XDGSCRAMClient{HashGeneratorFcn: SHA512} }
+		config.Net.SASL.Mechanism = sarama.SASLMechanism(sarama.SASLTypeSCRAMSHA512)
+	}
+
+	// kafka connect via SSL with PEM
+	if p.config.SaslSslEnabled {
+		certs := x509.NewCertPool()
+		pemPath := p.config.SaslPemPath
+		pemData, err := os.ReadFile(pemPath)
+		if err != nil {
+			p.logger.Fatalf("can't load cert: ", err.Error())
+		}
+		certs.AppendCertsFromPEM(pemData)
```

Comment:
Can we reuse this logic from elasticsearch output plugin:

https://github.com/ozontech/file.d/blob/master/plugin/output/elasticsearch/elasticsearch.go#L190-L198

It allows to pass path or cert content directly.
It will look like this:
```go
	if p.config.SaslSslEnabled {
		config.Net.TLS.Enable = true

		tlsCfg := tls.NewConfigBuilder()
		if err := tlsCfg.AppendCARoot(p.config.SaslPem); err != nil {
			p.logger.Fatalf("can't load cert: %s", err.Error())
		}
		config.Net.TLS.Config = tlsCfg.Build()
	}
```